src/App.css

```
@@ -5099,3 +5099,464 @@ body {
+}
+
+.hub-card-kv dt {
+  color: var(--muted);
+  min-width: 100px;
+  flex-shrink: 0;
+  font-size: 0.78rem;
+}
+
+.hub-card-kv dd {
+  margin: 0;
+  word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)